client/devdocs/index.js

```diff
@@ -63,15 +62,13 @@ export default class extends Component {
 								component ? (
 									componentName
 								) : (
-									<Link href={ getAdminLink( `?page=wc-admin&path=/devdocs/${ filePath }` ) }>
+									<Link href={ `admin.php?page=wc-admin&path=/devdocs/${ filePath }` }>
```

Comment:
What changed in `getAdminLink()` to make it unsuitable for this usage?

Reply:
In the `<Link />` component an `onClick` handler is added if the link type is `wc-admin`. When the link is clicked it appends the link href to the end of the base wc-admin URL. By using the function the onClick is using full URI.

The alternative change that would work is changing the link type in the getAdminLink() call.

Reply:
https://github.com/woocommerce/woocommerce-admin/blob/master/packages/components/src/link/index.js#L51:L53

Reply:
Oh, it looks like the `<Link>` component [internally calls `getAdminLink()`](https://github.com/woocommerce/woocommerce-admin/blob/master/packages/components/src/link/index.js#L41). 👍